docs/section10/10-11.md

```diff
@@ -0,0 +1,26 @@
+# Airside System - Rule 10-11  
+**Schema Version:** 0.0.12  
+**Mandatory Rule:** False 
+**Rule ID:** 10-11  
+**Rule Description:** Except for spaces with baseline system 9 or 10, if no cooling system exists or no cooling system has been submitted with design documents, the proposed building cooling system type shall be the same as modeled in the baseline building design and shall comply with the requirements of Section 6.  
+**Rule Assertion:** P-RMR = B-RMR   
+**Appendix G Section:** Table G3.1 Section 10 proposed column d.   
+**90.1 Section Reference:** None  
+
+**Data Lookup:** None  
+
+**Evaluation Context:** Evaluate each HeatingVentilationAirConditioningSystem Data Group
+
+**Applicability Checks:** None
+
+**Function Calls:**  
+1. get_proposed_hvac_modeled_with_virtual_cooling()
+
+## Rule Logic:  
+**Applicability Check 1:**  
+- Call function to get list of applicable HVAC systems due to virtual cooling being modeled in the proposed per G3.1-10 d:`applicable_HVAC_systems_cooling_list_p = get_proposed_hvac_modeled_with_virtual_cooling(U_RMR,P_RMR)`
+- For each hvac system in applicable_HVAC_systems_cooling_list_p: `for hvac_p in applicable_HVAC_systems_cooling_list_p:`    
+    **Rule Assertion:**
+    - Case 1: For applicable HVAC system: `outcome == "UNDETERMINED" and raise_message "Check that the proposed building system type is modeled the same system as is modeled in the baseline building design and that it complies with but does not exceed the requirements of Section 6."`"`  
```

Comment:
Please update message to point to indicate that cooling system should be checked, not just the "proposed system".